src/lib/containers/StackedRowHomebrew.js

```diff
@@ -59,54 +139,98 @@ export default class StackedRowHomebrew extends React.Component {
 
         let {data} = this.props
 
-        let x_data = []
-        data.facets.forEach(
-            item => {
-                if (item.facetType === "enumeration") {
-                    item.facetValues.forEach(
-                        facetValue => {
-                            if (item.columnName === "parentId") {
-                                x_data.push(facetValue.count)
+        let x_data = this.extractPropsData(data);
+        let total = 0
+
+        // sum up the counts of data
+        for (let key in x_data) { if (x_data.hasOwnProperty(key)) { total += x_data[key].count } }
+        let {colors} = this.state
+
+        return (
+            <div style={{marginBottom:"50px"}} className="container">
+                <div> 
+                    <span>
+                        <strong> {total} </strong> files shown by {this.props.alias}
+                    </span>
+                    <button 
+                        className="btn btn-default"
+                        type="button" 
+                        onClick={this.handleArrowClick(RIGHT_CLICK)} 
```

Comment:
Ohhh, I thought LEFT_CLICK and RIGHT_CLICK was going to be looking for the click modifier (you know, right-click!).  Suggest renaming to something else (PREVIOUS_ITEM_CLICK and NEXT_ITEM_CLICK?)